tests/test_metrics.py

```diff
@@ -396,6 +396,41 @@ def test_inventory_quantity_per_gwe():
     obs = metrics.inventory_quantity_per_gwe.func(inv, tsp)
     assert_frame_equal(exp, obs)
 
+	
+def test_transaction_quantity_per_gwe():
+    #exp is the expected output metrics
+    exp = pd.DataFrame(np.array([
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 1, 7, 3, 3,  10, 20, 'LWR Fuel', 'kg/GWe', 0.82),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 2, 8, 4, 3,  20, 30, 'FR Fuel', 'kg/GWe', 0.61),
+        (UUID('f22f2281-2464-420a-8325-37320fd418f8'), 3, 9, 5, 12, 30, 40, 'Spent Fuel', 'kg/GWe', 0.09),
+        ], dtype=ensure_dt_bytes([
+                ('SimId', 'O'), ('TransactionId', '<i8'), ('ResourceId', '<i8'),
+                ('ObjId', '<i8'), ('Time', '<i8'), ('SenderId', '<i8'),
+                ('ReceiverId', '<i8'), ('Commodity', 'O'), ('Units', 'O'),
+                ('Quantity', '<f8')]))
```

Comment:
could you put them all the argument inline ? I feel like this would be more readable...

Reply:
Are you referring to 'Quantity' only or everything starting from 'SimId'? If I put everything on 1 line, it would have more than 80 characters in 1 line. Will that be okay?

Reply:
all of them. I like to see them as:
```
('SimId', 'O'), 
('TransactionId', '<i8'), 
('ResourceId', '<i8'),
('ObjId', '<i8'), 
('Time', '<i8'), 
('SenderId', '<i8'),
('ReceiverId', '<i8'), 
('Commodity', 'O'), 
('Units', 'O'),
('Quantity', '<f8')]))
```